### PR TITLE
Title color contrast fix

### PR DESCRIPTION
**Pull request by blackfalcon**
# Alaska Airlines Pull Request

**Fixes:** #3 

## Summary:

This pull request addresses the issue related to issue #3, color contrast use in the auro-lockup. 

### BEFORE

Screen Shot 2021-03-17 at 8 15 58 PM

### AFTER

Screen Shot 2021-03-17 at 8 16 16 PM

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Comment by geoffrich** [comment]
@blackfalcon I assume we're waiting on design for this one?

**Comment by blackfalcon** [comment]
@vidalmenAS have you had a chance to review this? 
http://lockup-issue3.surge.sh/

File: demo/index.html [hunk 1]
```diff
@@ -7,16 +7,16 @@
     <link
       rel="stylesheet"
       type="text/css"
-      href="../node_modules/prismjs/themes/prism.css"
+      href="https://unpkg.com/prismjs@latest/themes/prism.css"
```
Comment (jason-capsule42) [inline demo/index.html:10]: Looks like prism js is getting updated about every 3 months. What do you think about locking this down to a known working version?

Comment (blackfalcon) [reply demo/index.html:10]: Looking through the commits, I don't see this being a huge issue. Also, this is the demo page in the repo, so very low risk if there are any changes.

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**